frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions.jsx

```diff
@@ -71,6 +73,10 @@ export default class ExpressionEditorSuggestions extends Component {
     this.props.onSuggestionMouseDown && this.props.onSuggestionMouseDown(index);
   }
 
+  onMouseDownCapture = _.memoize(i => {
```

Comment:
Nit: It's not a `onMouseDownCapture` handler, but a function returning these handlers, so I'd call it `createOnMouseDownCapture` or `getOnMuseDownCapture`.

Reply:
I agree, it's not really necessary for performance, but overall code after refactoring looks a bit better to me